### PR TITLE
MakeFile added

### PR DESCRIPTION
**Pull request by agarwalarjun123**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+build:
```
Comment (darshkpatel) [inline Makefile]: let's change build -> sense 
so the command is ``` make sense ```

Comment (darshkpatel) [inline Makefile]: let's change build -> sense 
so the command is ``` make sense ```

Comment (ba11b0y) [reply Makefile]: Ahh! I remember this :laughing:

**Comment by ba11b0y** [comment]
Tested! Merging!

**Review by ba11b0y — APPROVED** [review]

**Pull request merged**